assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/empty-mini-cart-contents-block/frontend.tsx

```diff
@@ -16,12 +17,22 @@ const EmptyMiniCartContentsBlock = ( {
 }: EmptyMiniCartContentsBlockProps ): JSX.Element | null => {
 	const { cartItems, cartIsLoading } = useStoreCart();
 
+	const elementRef = useRef< HTMLDivElement >( null );
+
+	useEffect( () => {
+		elementRef.current?.focus();
+	} );
+
 	if ( cartIsLoading || cartItems.length > 0 ) {
 		return null;
 	}
 
 	return (
-		<div className="wp-block-woocommerce-empty-mini-cart-contents-block">
+		<div
+			tabIndex={ 0 }
```

Comment:
We should consider using `-1` here - It allows focusing with JavaScript but wouldn't allow to focus using TAB and would not interrupt tabbing flow for accessibility.